org.eclipse.xtext.ui/src/org/eclipse/xtext/ui/editor/XtextSourceViewer.java

```diff
@@ -45,14 +49,28 @@ XtextSourceViewer createSourceViewer(Composite parent, IVerticalRuler ruler, IOv
 	
 	public static class DefaultFactory implements Factory {
 
+		/**
+		 * @since 2.19
+		 */
+		@Inject
+		private MembersInjector<XtextSourceViewer> membersInjector;
+		
 		@Override
 		public XtextSourceViewer createSourceViewer(Composite parent, IVerticalRuler ruler,
 				IOverviewRuler overviewRuler, boolean showsAnnotationOverview, int styles) {
-			return new XtextSourceViewer(parent, ruler, overviewRuler, showsAnnotationOverview, styles);
+			XtextSourceViewer result = new XtextSourceViewer(parent, ruler, overviewRuler, showsAnnotationOverview, styles);
+			membersInjector.injectMembers(result);
```

Comment:
is this a semantic change of the api of XtextSourceViewer

Reply:
It shouldn't be, since we do have a default value for the new field.